wdio/specs/news-item.js

```diff
@@ -0,0 +1,23 @@
+const assert = require('assert');
+
+describe('Municipality news item page', () => {
+
+  it('should show the news item with titles, date and body content in hebrew', () => {
+    browser.url('/municipality-1/node/54?language=he');
+    browser.waitForVisible('h2=News and Updates');
+    browser.waitForVisible('h3=מבצע סגירת חובות ארנונה לתושבים');
+    // Check that the date HTML tag exists on the page.
+    browser.waitForVisible('small');
```

Comment:
lets add a more explicit selector (e.g. `container > div > small`) as we might have `small` other places